daemon/api_sideload_n_try.go

```diff
@@ -157,7 +157,11 @@ func sideloadOrTrySnap(c *Command, body io.ReadCloser, boundary string, user *au
 	flags.RemoveSnapPath = true
 	flags.Unaliased = isTrue(form, "unaliased")
 	flags.IgnoreRunning = isTrue(form, "ignore-running")
-	flags.Transactional = isTrue(form, "transactional")
+	trasactionVals := form.Values["transaction"]
+	flags.Transaction = client.TransactionPerSnap
+	if len(trasactionVals) > 0 {
+		flags.Transaction = client.TransactionType(trasactionVals[0])
```

Comment:
this code and snapInstruction.validate should check that this has actually one of the 2 permitted values or it's indeed empty